### PR TITLE
tests(mixed): remove usage of `meta.props`

### PR DESCRIPTION
**Pull request by layershifter**
This PR completely removes usage of  `meta.props` from tests.

### CODE REVIEW

**Comment by layershifter** [comment]
@levithomason Any ideas why tests are failed on CI?

`27 01 2017 19:23:26.438:WARN [PhantomJS 2.1.1 (Linux 0.0.0)]: Disconnected (1 times), because no message in 10000 ms.`

**Comment by layershifter** [comment]
I've rebased to `master`, but still have no idea what is going with PhantomJS.

**Comment by levithomason** [comment]
I've marked this issue and will take a look soon as I can.

**Comment by layershifter** [comment]
I'll rebase to `master` today, there was updates in #1273 for `Popup` tests

**Comment by layershifter** [comment]
I've rebased, but, `Popup`'s tests still fail in same place.

**Comment by layershifter** [comment]
I'm making PR for `Popup`, I'll try to solve problem there. However, there is simple explanation why it worked: test used `_meta.props.positions` while they was defined in `_meta.props.positioning`, so it doesn't really work.

**Comment by layershifter** [comment]
Okay, problem is solved, need to rebase it 😄

**Comment by layershifter** [comment]
We need merge #1322 before this.

File: test/specs/commonTests.js [hunk 1]
```diff
@@ -486,27 +486,26 @@ export const rendersChildren = (Component, options = {}) => {
...
 // className from prop
 // ----------------------------------------
-const _noDefaultClassNameFromProp = (Component, propKey, options = {}) => {
+const _noDefaultClassNameFromProp = (Component, propKey, propValues, options = {}) => {
+  const { defaultProps = {} } = Component
   const { className = propKey, requiredProps = {} } = options
   // required props may include a prop that creates a className
   // if so, we cannot assert that it doesn't exist by default because it is required to exist
   // skip assertions for required props
+  if (propKey in defaultProps) return
```
Comment (layershifter) [inline test/specs/commonTests.js:495]: I've rebased, but tests failed on `Sidebar`, it has `defaultProps` for `direction`. So, I think it's expected behaviour there.

**Review by levithomason — APPROVED** [review]
Whew, great work here!  Very appreciative of you sticking to this and seeing it all the way through.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.65.0`

**Pull request merged**